pccommon/pccommon/config/containers.py

```diff
@@ -1,18 +1,20 @@
 from typing import Optional
 
-import orjson
 from pydantic import BaseModel
 
 from pccommon.tables import ModelTableService
-from pccommon.utils import orjson_dumps
 
 
 class ContainerConfig(BaseModel):
     has_cdn: bool = False
 
-    class Config:
-        json_loads = orjson.loads
-        json_dumps = orjson_dumps
+    # json_loads/json_dumps config have been removed
+    # the authors seems to indicate that parsing/serialization
+    # in Rust (pydantic-core) is fast (but maybe not as fast as orjson)
+    # https://github.com/pydantic/pydantic/discussions/6388
+    # class Config:
+    #     json_loads = orjson.loads
+    #     json_dumps = orjson_dumps
```

Comment:
we can't customize this with pydantic 2 but it seems that pydantic 2 is faster anyway